docs/contributing/contributing-rules.md

```diff
@@ -59,13 +59,30 @@ A well-written rule message includes:
 
 For an example of a good rule message, see: [this rule for Django's mark_safe](https://semgrep.dev/r?q=python.django.security.audit.avoid-mark-safe.avoid-mark-safe).
 
+Use the YAML multiline string operator `>-`. This operator enables you to write the rule message as one long line or multiple lines. Whichever way you choose, all will be interpreted as one line. This presents the best-looking rule message on the command-line.
```

Comment:
I actually preferred the other way of just giving the suggestion: 

"Use ....when rule messages span multiple lines. This presents the best-looking rule message in the command-line."

It was succinct and to the point.

if it's a single line, you can worry about escaping quotes and backslashes yourself

Reply:
That is true! Moving the conversation above: https://github.com/returntocorp/semgrep-docs/pull/551#discussion_r906347776